app/Http/Controllers/WorkoutLogController.php

```diff
@@ -35,7 +32,8 @@ public function createWorkoutLog (Request $request) {
         }
         $createdWorkoutLog = new WorkoutLog;
         $createdWorkoutLog->expertise_level = $request->expertiseLevel;
-        $createdWorkoutLog->deleted_date = $request->deletedDate;
+        $createdWorkoutLog->softDeletes();
```

Comment:
I think this creates a workout log, the softDelete should be called on other API/function